shared/modules/selectors/networks.ts

```diff
@@ -43,6 +44,60 @@ export function getSelectedNetworkClientId(
 ) {
   return state.metamask.selectedNetworkClientId;
 }
+/**
+ * Type to extend InternalNetworkState with multichain configurations.
+ */
+type ExtendedInternalNetworkState = InternalNetworkState & {
```

Comment:
any idea why `NetworkState` doesn't export `multichainNetworkConfigurationsByChainId`?

Reply:
oh... that's because it comes from the MultichainNetworkController.

https://github.com/MetaMask/core/blob/ec6967563ea6254f38ad14e81baa7a46a7b33636/packages/multichain-network-controller/src/types.ts#L84C13-L84C45

Reply:
working on this